src/core/flow.js

```diff
@@ -207,9 +210,12 @@ export function FlowWithPhases({
   if (endTurn === undefined) {
     endTurn = true;
   }
-  if (undo === undefined) {
+  if (undo === undefined && undoableMoves === undefined) {
```

Comment:
How about we just get rid of the `undo` boolean? We can use the following semantics instead:
- To make all moves undoable:
`undoableMoves === undefined`

- To make no move undoable:
`undoableMoves === []`

- To make some moves undoable:
`undoableMoves === [ ... ]`

Reply:
Once we adopt these semantics, I think that will also pave the way to having consistency between `undoableMoves` and `allowedMoves`. For example, we can implement both at a global and per-phase level easily.

Reply:
Yes I like it a lot, but was a bit reluctant to remove ‘undo‘ without consent. 
So basically we come with undo/redo out of the box and a developer would have to restrict it, if it's not needed. Sounds great to me!